### PR TITLE
MON-3249: Update telemeter-client to allow TLS through rbac proxy

### PR DESCRIPTION
**Pull request by marioferh**
Update telemeter-client deployment to work with new feature TLS.

Update telemeter branch to get updated json
Generate new jsonnetfile with new branch
Make generate to update deployment.yaml

Depends on:
https://github.com/openshift/cluster-monitoring-operator/pull/1904 
https://github.com/openshift/telemeter/pull/455

* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: assets/telemeter-client/service.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   annotations:
-    service.beta.openshift.io/serving-cert-secret-name: telemeter-client-tls
+    service.beta.openshift.io/serving-cert-secret-name: telemeter-client-certs
```
[the remark this answers is not in the document]
Comment (marioferh) [reply assets/telemeter-client/service.yaml:5]: Sorry wrong answer, you are right the old secret is still there if we apply new config:
➜  ~ oc get secrets -n openshift-monitoring  | grep telemeter-client-certs
telemeter-client-certs                                  kubernetes.io/tls                     2      85s
➜  ~ oc get secrets -n openshift-monitoring  | grep telemeter-client-tls  
telemeter-client-tls                                    kubernetes.io/tls                     2      91m
➜  ~

**Comment by marioferh** [comment]
Updated with last telemeter fix: https://github.com/openshift/telemeter/pull/472
@simonpasquier

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by simonpasquier** [comment]
/label  docs-approved
/label px-approved

no functional change.

**Pull request merged**